nss-day-cohort-41/react-nutshell-the-hazelnuts
# Created homepage links

# Description
- Home page now has links directly from home page, substitute for navbar until user reaches any of the subpages.

## Type of change


# Testing Instructions
- Populate and serve json file with "articles" subpage, and key/value pairs from example.json
- login with any email and password to land on home page
- Click each link to ensure they direct to the correct subpages.

# Checklist:
